README.md

```diff
@@ -60,77 +61,66 @@ export ARTIFACTORY_USERNAME=firstname.lastname
 export ARTIFACTORY_PASSWORD=Long_string_of_gibberish_abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUV
 ```
 
-- `ARTIFACTORY_USERNAME`: 
+- `ARTIFACTORY_USERNAME`:
   Find your username in [JFrog](https://digitalasset.jfrog.io/ui/admin/artifactory/user_profile); usually `firstname.lastname`.
-- `ARTIFACTORY_PASSWORD`: 
+- `ARTIFACTORY_PASSWORD`:
   This is your **API Key**, *NOT* your password. Find your API key at the bottom of **Authentication Settings** in your profile.
 
-## Build and view the docs locally
-
-`cd` to the `docs` directory in the repository:
-
-```zsh
-cd docs
-```
-
 ### Download docs
 
 Run the following script to download the documentation tarballs from the Daml and Canton repos:
 
 ```zsh
-./download.sh
+download
 ```
 
 ### Live preview
 
-Run the `live-preview.sh` script to render a local view of the site. The html files are in `docs/workdir/build`. Rerun the script to view any edits to `docs/index/` files.
-
-:warning: If you make changes to docs managed by the other repos, you have to commit them there. You can only make changes to the top-level `index.rst` files, containing the master TOC, that live in this repository. 
-
-:warning: If you do make a change to the TOCs in this repo, you then have to duplicate the change in the relevant satellite repo.
+Run the `live-preview` script to render a local view of the site.
 
 ```zsh
-./live-preview.sh
+live-preview
 ```
 
+The html files are in `workdir/build/gen`. Rerun the script to view any edits to `docs/` files or the top-level ToC (`_toc.yml`).
```

Comment:
Rerunning the script should ideally not be necessary here but I'll leave that here for the time being. Please test and update accordingly in a follow-up PR if needed.